src/credentials/GroupDataProviderImpl.cpp

```diff
@@ -717,14 +717,14 @@ struct KeySetData : PersistentData<kPersistentBufferMax>
     chip::KeysetId prev            = 0xffff;
     bool first                     = true;
 
-    uint16_t keyset_id            = 0;
-    KeySet::SecurityPolicy policy = KeySet::SecurityPolicy::kStandard;
```

Comment:
As a followup: please rename the security policy enum values to match the spec in its state after https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/4890 merges.

Reply:
Filed issue.